[2024-08-31T17:28:05.351+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T00:00:00+00:00 [queued]>
[2024-08-31T17:28:05.367+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T00:00:00+00:00 [queued]>
[2024-08-31T17:28:05.368+0000] {taskinstance.py:2170} INFO - Starting attempt 19 of 24
[2024-08-31T17:28:05.400+0000] {taskinstance.py:2191} INFO - Executing <Task(SqlToS3Operator): extract_hr> on 2024-08-30 00:00:00+00:00
[2024-08-31T17:28:05.412+0000] {standard_task_runner.py:60} INFO - Started process 255 to run task
[2024-08-31T17:28:05.418+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ETL_Dag_S3', 'extract_hr', 'scheduled__2024-08-30T00:00:00+00:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag_s3.py', '--cfg-path', '/tmp/tmpducsj893']
[2024-08-31T17:28:05.422+0000] {standard_task_runner.py:88} INFO - Job 260: Subtask extract_hr
[2024-08-31T17:28:05.507+0000] {task_command.py:423} INFO - Running <TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T00:00:00+00:00 [running]> on host f6a0ecf2184f
[2024-08-31T17:28:05.675+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jasontran' AIRFLOW_CTX_DAG_ID='ETL_Dag_S3' AIRFLOW_CTX_TASK_ID='extract_hr' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='19' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-30T00:00:00+00:00'
[2024-08-31T17:28:05.695+0000] {base.py:83} INFO - Using connection ID 'PostgreSQL_conn' for task execution.
[2024-08-31T17:28:05.710+0000] {base.py:83} INFO - Using connection ID 'PostgreSQL_conn' for task execution.
[2024-08-31T17:28:05.801+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:221 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-08-31T17:28:05.924+0000] {sql_to_s3.py:179} INFO - Data from SQL obtained
[2024-08-31T17:28:05.936+0000] {sql_to_s3.py:186} INFO - Writing data to temp file
[2024-08-31T17:28:06.003+0000] {sql_to_s3.py:189} INFO - Uploading data to S3
[2024-08-31T17:28:06.029+0000] {base.py:83} INFO - Using connection ID 'AWS_S3_conn' for task execution.
[2024-08-31T17:28:06.033+0000] {connection_wrapper.py:381} INFO - AWS Connection (conn_id='AWS_S3_conn', conn_type='aws') credentials retrieved from extra.
[2024-08-31T17:28:07.750+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ETL_Dag_S3, task_id=extract_hr, execution_date=20240830T000000, start_date=20240831T172805, end_date=20240831T172807
[2024-08-31T17:28:07.806+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-31T17:28:07.852+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
